src/test/scala/is/hail/methods/SkatSuite.scala

```diff
@@ -163,7 +162,7 @@ class SkatSuite extends SparkSuite {
       .annotateSamples(TFloat64(), List("cov", "Cov1"), s => cov1Array(s.asInstanceOf[String].toInt))
       .annotateSamples(TFloat64(), List("cov", "Cov2"), s => cov2Array(s.asInstanceOf[String].toInt))
       .annotateSamples(TBoolean(), List("pheno"), s => phenoArray(s.asInstanceOf[String].toInt))
-      .annotateVariantsExpr("va.genes = [v.start % 2, v.start % 3].toSet") // three overlapping genes
+      .annotateVariantsExpr("va.gene = v.start % 3") // three genes
```

Comment:
Same as comment on other tests: shouldn't we test the product/explode/skat use case?